controllers/topics.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectTopics } = require('../models/topics.model');
+
+exports.getTopics = (req, res) => {
+    selectTopics()
+        .then(response => {
+            res.status(200).send({ topics: response });
+        }).catch(err => {
```

Comment:
this .catch will only be triggered if the there is a problem with resolving the promise - if a topic is not found, in the response received by the model, the model will just get an empty array which it will send to this .then() and then that will be sent in the res.send. Where might you need to move this 404 handling logic?